metadata.yaml

```diff
@@ -43,6 +43,13 @@ provides:
 requires:
   alertmanager:
     interface: alertmanager_dispatch
+  ingress-per-unit:
```

Comment:
Please change the relation name to `ingress`, it’s clearer to the Juju admin and will remain consistent with prometheus-k8s and likely with future distributed loki (even though that will use a different interface)